css/properties/alignment-baseline.json

```diff
@@ -98,7 +98,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "≤5.1"
```

Comment:
Note that the default value seems to have changed from `auto` to `baseline` in Safari 18.1.